react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { id: null };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    const number = Number(event.target.className);
+    const openNote = number === this.state.id ? null : number;
+    this.setState({ id: openNote });
+  }
+
+  render() {
+    return (
+      this.props.topics.map((topic, index) => {
+        return (
+          <div key={index}>
```

Comment:
For the `key` you want to avoid using `index`, it can lead to issues. Use `id` when it is available:
```JSX
<div key={topic.id}>
```